src/client_backend/triton_c_api/shared_memory_manager.cc

```diff
@@ -42,7 +42,6 @@ SharedMemoryManager::~SharedMemoryManager()
   UnregisterAll(TRITONSERVER_MEMORY_GPU);
 }
 
-#ifdef TRITON_ENABLE_GPU
```

Comment:
Several instances of functions like this were `#ifdef TRITON_ENABLE_GPU`'d out even though they didn't reference any types specific to `<cuda_runtime_api.h>`, so I removed the unnecessary conditional compilation.